static/templates/settings/organization-settings-admin.handlebars

```diff
@@ -12,99 +12,117 @@
+            title="{{t 'If checked, users can edit the content and topics of their old messages.' }}"
+            class="inline-block">
+            {{t "Users can edit old messages" }}
+          </label>
+        </div>
+        <div class="input-group disableable {{#unless realm_allow_message_editing}}control-label-disabled{{/unless}}">
+          <label for="realm_message_content_edit_limit_minutes"
+            id="id_realm_message_content_edit_limit_minutes_label"
+            title="{{t 'If non-zero, users can edit their message for this many minutes after it is sent. If zero, users can edit all their past messages.' }}">
+            {{t 'Message edit limit in minutes (0 for no limit)' }}
+          </label>
+          <input type="text" id="id_realm_message_content_edit_limit_minutes"
+                 name="realm_message_content_edit_limit_minutes"
+                 class="admin-realm-message-content-edit-limit-minutes"
+                 value="{{ realm_message_content_edit_limit_minutes }}"
+            {{#unless realm_allow_message_editing}}disabled="disabled"{{/unless}} />
+        </div>
+        <div class="input-group">
+          <label for="realm_default_language">{{t "Default language" }}:</label>
+          <select name="realm_default_language" id="id_realm_default_language">
+            {{#each language_list}}
+            <option value='{{this.code}}'>{{this.name}}</option>
+            {{/each}}
+          </select>
+        </div>
+        <div class="input-group">
+            <label for="realm_waiting_period_threshold">{{t "Waiting period for stream creation (in days)" }}</label>
+            <input type="text" id="id_realm_waiting_period_threshold"
+                 name="realm_waiting_period_threshold"
+                 class="admin-realm-message-content-edit-limit-minutes"
+                 value="{{ realm_waiting_period_threshold }}"/>
+        </div>
+        <div class="input-group organization-submission">
+          <input type="submit" class="button" value="{{t 'Save changes' }}" />
+        </div>
     </div>
+      <div class="realm-icon-section box-shadow border-radius">
+          <div class="inline-block">
+              <img id="realm-settings-icon" src="{{ realm_icon }}"/>
+              <div id="realm_icon_file_input_error" class="text-error"></div>
+              <input type="file" name="realm_icon_file_input" class="notvisible"
+                     id="realm_icon_file_input" value="{{t 'Upload icon' }}"/>
+              <div id="upload_icon_spinner"></div>
+          </div>
+          <div class="inline-block">
+              <button class="button sea-green w-200 m-t-20 block input-size"
+                      id="realm_icon_upload_button">{{t 'Upload New Icon' }}</button>
+              <div class="w-200 dark-grey center m-t-20">
+                  Upload a small (1024x1024) icon for realm here.
+              </div>
+          </div>
+      </div>
```

Comment:
This template change is super hard to read; can you split out a preliminary commit doing any reindenting from the actual additions you want to make?